spec/helpers/boards_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the BoardsHelper. For example:
+#
+# describe BoardsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe BoardsHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Could you remove this line? It'll generate a reminder about pending tests every RSpec run, which we don't really need since we won't actually be adding tests here.